app/views/maths_and_physics/claims/teaching_maths_or_physics.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for(:page_title, page_title(t("maths_and_physics.questions.teaching_maths_or_physics"), policy: current_policy_routing_name, show_error: current_claim.errors.any?)) %>
+<% path_for_form = current_claim.persisted? ? claim_path(current_policy_routing_name) : claims_path(current_policy_routing_name) %>
```

Comment:
This was so subtle it confused me for a while!

Reply:
yeah, it is subtle... any suggestions for how to make it less confusing?

Reply:
Not sure, I think it's one of the Rails things you need to get used to decoding.